app/models/client_opportunity_match.rb

```diff
@@ -491,7 +496,9 @@ def succeeded!
       if route.should_cancel_other_matches
         client_related_matches.each do |match|
           if match.current_decision.present?
-            MatchEvents::DecisionAction.create(match_id: match.id,
+            MatchEvents::DecisionAction.create(
```

Comment:
Same as with `canceled_events` if we add a `has_many: :decision_action_events` we can take advantage of the relationship to create the event.